BlackNectar/BlackNectar/FilterViewController.swift

```diff
@@ -216,16 +262,21 @@ extension FilterViewController {
     
 }
 
-//MARK: Loads Stores When User Pans
+//MARK: Style Menu Code
 extension FilterViewController {
     
-    func mapView(_ mapView: MKMapView, regionDidChangeAnimated animated: Bool) {
-        
-        let center = mapView.centerCoordinate
+    func styleButtonOn(button: UIButton) {
         
-        LOG.debug("User dragged Map Screen to: \(center)")
+        button.backgroundColor = Colors.fromRGB(red: 235, green: 191, blue: 77)
+        button.titleLabel?.font = UIFont(name: "Oxygen-Bold", size: 16)
+        UIView.transition(with: button, duration: 0.4, options: .transitionCrossDissolve, animations: nil, completion: nil)
```

Comment:
Technically....the code that changes the buttons should be in an animations closure. That closure would then be passed to the UIView.transition method.

```swift
let animations = {
    button.backgroundColor = Colors.fromRGB(red: 235, green: 191, blue: 77)
    button.titleLabel?.font = UIFont(name: "Oxygen-Bold", size: 16)
}
UIView.transition(with: button, duration: 0.4, options: .transitionCrossDissolve, animations: animations, completion: nil)
```